web/geosearch/datapunt_geosearch/datasource.py

```diff
@@ -161,6 +162,11 @@ def get_variable_sql(self, table: str) -> Dict[str, sql.Identifier]:
             ),
         )
 
+    def get_db_crs(self) -> sql.Literal:
+        if self.crs is not None:
+            return sql.SQL(self.crs.split(":")[-1])
+        return sql.SQL("28992")
```

Comment:
Nitpick, but putting the exceptional case in the `if`-clause means your code is flatter (esp. if you have to add more exceptions later on). Also, a comment explaining the expected format would be nice:

```suggestion
        if self.crs is None:
            return sql.SQL("28992")
        # "EPSG:28992" -> "28992"
        return sql.SQL(self.crs.split(":")[-1])
```